activerecord/lib/active_record/connection_adapters/abstract/quoting.rb

```diff
@@ -150,6 +150,10 @@ def quoted_time(value) # :nodoc:
         quoted_date(value).sub(/\A2000-01-01 /, "")
       end
 
+      def quoted_binary(value) # :nodoc:
```

Comment:
Some quoting methods (e.g. `quote_string`, `quoted_date`, `quoted_time`, etc) are public.
Should this method be private?